config/passport.js

```diff
@@ -0,0 +1,75 @@
+const passport = require('passport');
+const JwtStrategy = require('passport-jwt').Strategy;
+const ExtractJwt = require('passport-jwt').ExtractJwt;
+const mongoose = require('mongoose');
+var user = require('./../models/user');
```

Comment:
I would recommend only using let and const for variables just so you wont have to worry about any hoisting with var.